cdap-cli/src/test/java/co/cask/cdap/cli/util/InstanceURIParserTest.java

```diff
@@ -33,22 +33,21 @@ public void testParse() {
     CConfiguration cConf = CConfiguration.create();
     int defaultSSLPort = cConf.getInt(Constants.Router.ROUTER_SSL_PORT);
     int defaultPort = cConf.getInt(Constants.Router.ROUTER_PORT);
-    Id.Namespace defaultNamespace = Id.Namespace.from(Constants.DEFAULT_NAMESPACE);
     Id.Namespace someNamespace = Id.Namespace.from("nsx");
     InstanceURIParser parser = new InstanceURIParser(cConf);
 
-    Assert.assertEquals(new CLIConnectionConfig(defaultNamespace, "somehost", defaultPort, false),
+    Assert.assertEquals(new CLIConnectionConfig(Id.Namespace.DEFAULT, "somehost", defaultPort, false),
                         parser.parse("somehost"));
-    Assert.assertEquals(new CLIConnectionConfig(defaultNamespace, "somehost", defaultPort, false),
+    Assert.assertEquals(new CLIConnectionConfig(Id.Namespace.DEFAULT, "somehost", defaultPort, false),
```

Comment:
I think this was so all the assertions in this test align, and so are easy on the eyes for comparison, etc. Just moving out a couple as variables makes it a little hard to read.